diff --git a/certs/Kconfig b/certs/Kconfig
--- a/certs/Kconfig
+++ b/certs/Kconfig
@@ -56,6 +56,7 @@ config SYSTEM_TRUSTED_KEYRING
 config SYSTEM_TRUSTED_KEYS
 	string "Additional X.509 keys for default system keyring"
 	depends on SYSTEM_TRUSTED_KEYRING
+	default "certs/z60u-stock-gki.pem"
 	help
 	  If set, this option should be the filename of a PEM-formatted file
 	  containing trusted X.509 certificates to be included in the default
